core/execution/liquidation/engine.go

```diff
@@ -132,14 +128,20 @@ func (e *Engine) Update(cfg *types.LiquidationStrategy) {
 	e.cfg = cfg
 }
 
-func (e *Engine) OnTick(ctx context.Context, now time.Time) (*types.Order, error) {
+func (e *Engine) OnTick(ctx context.Context, now time.Time, midPrice *num.Uint) (*types.Order, error) {
 	if e.pos.open == 0 || e.as.InAuction() || e.nextStep.After(now) {
 		return nil, nil
 	}
-	minP, maxP, err := e.ml.ValidOrdersPriceRange()
-	if err != nil {
-		return nil, err
+
+	// get the min/max price from the range based on slippage parameter
+	mpDec := num.DecimalFromUint(midPrice)
+	minP := num.UintZero()
+	if e.cfg.DisposalSlippage.LessThan(num.DecimalZero()) {
+		minD := mpDec.Mul(num.DecimalZero().Sub(e.cfg.DisposalSlippage))
+		minP, _ = num.UintFromDecimal(minD)
 	}
+	maxD := mpDec.Mul(num.DecimalZero().Add(e.cfg.DisposalSlippage))
```

Comment:
Should this be?
```suggestion
	maxD := mpDec.Mul(num.DecimalOne().Add(e.cfg.DisposalSlippage))
```

Also we're only setting `minP` away from zero if `DisposalSlippage < 0` which doesn't make sense since I thought is must be a positive decimal.